lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -1970,7 +1970,7 @@ void mme_app_handle_initial_context_setup_failure(
      * the UE context.
      */
     ue_context_p->ue_context_rel_cause = S1AP_INITIAL_CONTEXT_SETUP_FAILED;
-    if (ue_context_p->mm_state == UE_UNREGISTERED) {
+    if (ue_context_p->mm_state != UE_REGISTERED) {
```

Comment:
mm_state_t used within structure, ue_mm_context_s has only two states. i.e as defined below
 79 
 typedef enum {
 80   UE_UNREGISTERED = 0,
 81   UE_REGISTERED,
 82 } mm_state_t;
 
 Both checks results into same results.

Reply:
Ahh good point, I was confusing this with the FSM state.